config/config.sample-complete.yaml

```diff
@@ -7,6 +7,9 @@ homeserver:
 
 username_prefix: "slack_"
 
+
+RMAU_limit: 100
```

Comment:
We tend to go lowercase with these. I'm also wondering if we should have co-located it with the room limits, but it's probably a bit late for that now.

Reply:
I thought I was keeping it consistent with something, but apparently it was the exact opposite. Fixed in 121e102